glossary.yml

```diff
@@ -5726,6 +5726,10 @@
     term: ግጥም ጥርብ ግራፍ
     def: >
       የቁጥር ውሂብ ስብስብ ስርጭት ግራፊክ ውክልና ፣ ብዙውን ጊዜ ቀጥ ያለ የአሞሌ ግራፍ
+  uk:
+    term: "гістограма"
+    def: >
+      Спосіб графічного представлення табличних даних; кількісні співвідношення деякого показника представлені у вигляді прямокутників, площі яких пропорційні.
```

Comment:
"Графічне представлення розподілу числових даних, де дані згруповані у рівновіддалені інтервали, а висота кожної групи відповідає частоті точок даних у цьому інтервалі."

What do you think about this definition?

Reply:
@mariyaspatar what do you think?

Reply:
@mariyaspatar do you have plans to finish this PR soon? If you don't have time right now, that's completely understandable - one option would be to add by agreement one or more of @VeronikaShevc @YehorBoiar @Herobread to your fork as collaborators (see https://ukrainian-carpentries.github.io/git-novice/08-collab.html for instructions) and then they can push new commits to your branch and update your PR. Please let us know if you have any difficulties with completing this PR, and we will be happy to help!

Reply:
Hi all!
I changed the translation for the term, hopefully it has been updated. Please let me know if there is any issue

Reply:
Thanks @mariyaspatar - the translation of "histogram" is updated. I have also noticed the three new translations, but just now - they also need review, and fixing conflicts. Maybe it would have been easier if that was done in separate pull requests, one per term, for the start. @YehorBoiar @Herobread could you please review the updated and the new translations - would be nice if @mariyaspatar would have at least one translation incorporated into the public website before July 22nd official finish of the STEP project.